packages/tatum-neo/src/index.ts

```diff
@@ -2,6 +2,7 @@ export * from './offchain'
 export * from './record'
 export * from './tatum'
 export * from './wallet'
+export * from './blockchain'
```

Comment:
Nit: when I go around files like this, I try to sort the imports alphabetically, for convenience of someone reading it in the future. Not necessary, but I think you could find it useful.

Reply:
I can change it, no problem :)